src/lib/components/home/artwork_rails/artwork_rail.tsx

```diff
@@ -207,13 +207,21 @@ class ArtworkRail extends React.Component<Props & RelayPropsWorkaround, State> {
     if (this.state.loadFailed) {
       return null
     }
+    const hasFetched = this.props.relay && this.props.relay.variables.fetchContent
 
     const sideMargin = isPad ? 40 : 20
     const style: any = { marginLeft: sideMargin, marginRight: sideMargin }
-    if (!(this.props.relay && this.props.relay.variables.fetchContent)) {
+    if (!hasFetched) {
       // if there's no content, set minHeight to prevent spinners from all the rails showing
       style.minHeight = 400
     }
+
+    const hasArtworks = hasFetched && this.props.rail.results && this.props.rail.results.length
+    if (!hasArtworks) {
```

Comment:
I think my variable name is misleading - `hasFetched` is actually more like `hasStartedFetching`. It's set [here](https://github.com/artsy/emission/blob/master/src/lib/components/home/artwork_rails/artwork_rail.tsx#L63). I can change the name so it makes more sense.